microsoft/botframework-solutions
# [Skill Template][TypeScript] Add models and resources

## Description

**Depends** on #947

- Add Cognitive Models
- Add Deployment Scripts
- Add Response Resources for each dialog
- Add PowerShell scripts

Related to #941

## Testing Steps

1. Go to `AI/templates/Skill-Template/src/typescript/generator-botbuilder-skill/generators/app/templates/customSkill/cognitiveModels`
1.1 Go to `AI/templates/Skill-Template/src/typescript/generator-botbuilder-skill/generators/app/templates/customSkill/deploymentScripts`
1.2 Go to `AI/templates/Skill-Template/src/typescript/generator-botbuilder-skill/generators/app/templates/customSkill/src/dialogs`
2. This change structure should be similar to C# solution
